hw_38_router/src/pages/Main.jsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+
+import Card from '../components/Card.jsx';
+import Button from '../components/Button.jsx';
```

Comment:
the good practice is creating index.jsx file if folder, for instance components, and export all components form it,
export {Card} from './Card.jsx'
export {Button} from './Button.jsx'
you did it for pages, same approach